requirements.txt

```diff
@@ -2,9 +2,9 @@ numpy>=1.11
 sympy==1.1
 scipy
 matplotlib
-pytest
+pytest-runner
```

Comment:
?

Reply:
For `python setup.py test` needs standalone pytest